app/views/hosts/_list.html.erb

```diff
@@ -31,8 +33,16 @@
         </td>
         <td class="hidden-xs ellipsis"><%= (icon(host.operatingsystem, :size => "16x16") + " #{host.operatingsystem.to_label}").html_safe if host.operatingsystem %></td>
         <td class="hidden-tablet hidden-xs ellipsis"><%= host.compute_resource_or_model %></td>
+        <td class="hidden-tablet hidden-xs ellipsis">
+          <% if host.owner_type == 'User' %>
```

Comment:
these 5 lines should ideally be extracted as a helper and called directly from here